### ISSUE TITLE
Using conventionalcommits as `changelogPreset` causes `lerna version` to do nothing

### ISSUE THREAD

**Issue by jlipps** [lerna/lerna#3871]
## Current Behavior

[Appium](https://github.com/appium/appium) has a `lerna.json` that has the following field:

```json
{
  "changelogPreset": "conventional-changelog-conventionalcommits"
}
```

Up until the v7 release of conventional-changelog, this worked as expected. After upgrading to v7+, the `lerna version` process exits immediately (with zero exit code) after loading the conventionalcommits package, rather than proceeding with the versioning process.

I originally reported this to the other project as https://github.com/conventional-changelog/conventional-changelog/issues/1157, but the maintainers there mentioned it is up to Lerna to update its use of conventionalcommits to reflect the breaking changes that have happened there.

I'm currently working around this issue by keeping the version of `conventional-changelog-conventionalcommits` pinned to `6.1.0`.

## Expected Behavior

I expect that using this `changelogPreset` setting will result in the version suggestion dialog instead of Lerna exiting.

## Steps to Reproduce

* Clone https://github.com/appium/appium.git
* `npm install`
* `npm install --save conventional-changelog-conventionalcommits@latest`
* `npx lerna version`

### Environment

```
lerna notice cli v7.4.1
lerna info versioning independent

 Environment info:

  System:
    OS: macOS 12.6.7
    CPU: (10) arm64 Apple M1 Pro
  Binaries:
    Node: 18.17.0 - ~/.nvm/versions/node/v18.17.0/bin/node
    Yarn: 1.22.19 - ~/Code/gh/appium/appium/node_modules/.bin/yarn
    npm: 10.1.0 - ~/.nvm/versions/node/v18.17.0/bin/npm
  Utilities:
    Git: 2.39.0 - /opt/homebrew/bin/git
  npmPackages:
    lerna: 7.4.1 => 7.4.1
```

**Comment by kerryj89** [comment]
Thanks for writing up the issue.  I've been having isolated issues so I thought it was just me again.  Surely we aren't the only two running into this issue?

**Comment by fahslaj** [comment]
Thank you for reporting this issue, and thank you to @amorscher for fixing it with PR #3873 . I will follow up here as soon as the patch has been released in a new version of Lerna.

**Comment by fahslaj** [comment]
This has been released with Lerna 7.4.2

**Comment by yhuard** [comment]
Hi @fahslaj, it seems that 7.4.2 wasn't published to npm, the latest version on https://www.npmjs.com/package/lerna still is 7.4.1. Can you have a look please? I'm having the same problem as reported on this issue :)

**Comment by yhuard** [comment]
You're absolutely right @amorscher, thank you! The new version didn't pop up when I ran `yarn upgrade-interactive --latest` and it doesn't show on https://www.npmjs.com/package/lerna since it's not tagged as `latest`, but I was able to pull the version 7.4.2 nonetheless. And it does indeed fix the problem, congrats 🙌

**Comment by fahslaj** [comment]
Thank you for letting me know @yhuard . This is now fixed and 7.4.2 is properly tagged as `latest`.
